BepInEx.IL2CPP/Preloader.cs

```diff
@@ -83,5 +86,48 @@ public static void Run()
                 throw;
             }
         }
+
+        private static void InitializeUnityVersion()
+        {
+            try
+            {
+                ConfigEntry<String> config = ConfigFile.CoreConfig.Bind(
+                    "Unity", "Version",
+                    String.Empty,
+                    "Allows you to manually set the Unity version .");
```

Comment:
I suggest moving this to a static readonly field as we do with other config values.

Moreover, I think the config option should be described better as its use is mainly for Il2Cpp. I suggest the following values:

* Section: `IL2CPP`
* Name: `UnityVersion`
* Description: `Unity version to report to Il2CppUnhollower. If empty, version is automatically determined from the game process.`